src/Shared/LogMessagePacketBase.cs

```diff
@@ -1218,6 +1241,16 @@ private BuildMessageEventArgs ReadBuildMessageEventFromStream(ITranslator transl
             return buildEvent;
         }
 
+        private ResponseFileUsedEventArgs ReadResponseFileUsedEventFromStream(ITranslator translator, string message, string helpKeyword, string senderName)
+        {
+            MessageImportance importance = MessageImportance.Normal;
+            translator.TranslateEnum(ref importance, (int)importance);
```

Comment:
I'm a little confused by the mismatches here between ReadResponseFileUsedEventFromStream and WriteResponseFileUsedEventToStream. Writing only ever writes the response file path, whereas this first reads the importance then reads the response file path. That feels wrong. Is it correct?